docs/DeveloperGuide.md

```diff
@@ -1044,16 +1044,16 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
These are intentional; it saves the trouble of having to maintain the numberings in the case edits are made as they are automatically converted into the appropriate numbering when deployed; do kindly revert unless there is a good reason not to.

Reply:
I do these changes because our numbering is very messy initially

Reply:
I don't follow, the numbering w.r.t. actual number are all 1., as this would automatically be assigned the relevant numbers based on indentation and so on; if you have doubts, there is a md preview option in intelliJ for you to verify this behavior.

As before, I would recommend reverting these changes